auth-web/src/components/auth/manage-business/AffiliatedEntityList.vue

```diff
@@ -111,6 +170,12 @@ export default class AffiliatedEntityList extends Vue {
     return corpType === CorpType.NAME_REQUEST
   }
 
+  private isApprovedForIA (business: Business): boolean {
+    return this.isNameRequest(business.corpType.code) &&
+      business.nameRequest?.state === NrState.APPROVED &&
+      LaunchDarklyService.getFlag(LDFlags.IaSupportedEntities)?.includes(business.nameRequest?.legalType)
```

Comment:
shouldn't this be using the new attributes Thor added on NR GET response instead of checking against a configured value?

Reply:
I thought i heard something about this but in a GET on `https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v1/nameRequests/<nrNumber>` we currently only do a proxy request through LEAR->NAMEX and back, i didn't see any additional properties. 
Where should i look for these?

Reply:
Not sure if it is implemented yet. If not in place, I guess it's good to go with configured value.

Reply:
Totally, i believe once it is implemented we would replace the first two properties but the third (feature flag) would most likely still be required for control on what we allow as per environment. 

Thanks for bringing this up, will look into these new properties and where to find them 👍